bus/routes.go

```diff
@@ -1779,14 +1783,22 @@ func (b *Bus) paramsHandlerUploadGET(jc jape.Context) {
 	})
 }
 
-func (b *Bus) consensusState() api.ConsensusState {
-	cs := b.cm.TipState()
+func (b *Bus) consensusState(ctx context.Context) (api.ConsensusState, error) {
+	index, err := b.cs.ChainIndex(ctx)
+	if err != nil {
+		return api.ConsensusState{}, err
+	}
 
-	return api.ConsensusState{
-		BlockHeight:   cs.Index.Height,
-		LastBlockTime: api.TimeRFC3339(cs.PrevTimestamps[0]),
-		Synced:        time.Since(cs.PrevTimestamps[0]) <= 3*time.Hour,
+	var synced bool
+	block, found := b.cm.Block(index.ID)
```

Comment:
Not sure how that happened. This was my mistake but I realised it right after opening the PR and then I closed it. I even left a comment explaining why https://github.com/SiaFoundation/renterd/pull/1797#issuecomment-2587917525

Reply:
Oh. I see what happened. Nate was probably on my branch when he made the auth changes and didn't notice.